man/git-ftp.1.md

```diff
@@ -189,6 +189,26 @@ But, there is not just FTP. Supported protocols are:
 `ftpes://...`
 :	FTP over explicit SSL (FTPES) protocol
 
+# EXAMPLES
+
+Upload your files to an FTP server the first time:
+
+	$ git ftp init -u "$USER" -P "ftp://example.com/public_html"
+
+It will authenticate with the username stored in `$USER` and ask for the
+password. Using SFTP is a much better option. But be aware of the different
+handling of relative and absolute paths. If the directory `public_html` is in
+the home directory on the server, then upload like this:
+
+	$ git ftp init -u "$USER" --key "$HOME/.ssh/id_rsa" "sftp://example.com/~/public_html"
+
+Otherwise it will use an absolute path, for example:
+
+	$ git ftp init -u "$USER" --key "$HOME/.ssh/id_rsa" "sftp://example.com/var/www"
+
+It is quite common to deactivate server certificate checking with the
+`--insecure` option.
```

Comment:
What do you think of: "It is quite common to deactivate server certificate checking with the`--insecure` option, especially if you want use the private key without the public key."